findbugs/build.gradle

```diff
@@ -286,19 +204,6 @@ publishing {
         classifier 'javadoc'
       }
     }
-
-    annotations(MavenPublication) {
```

Comment:
we do need publishing config / maven + signing plugins in the new project to publish annotations artifacts.